src/state/layout.ts

```diff
@@ -1,36 +1,51 @@
 import { createSlice, type PayloadAction } from '@reduxjs/toolkit';
 
 import { TradeLayouts } from '@/constants/layout';
-import { LocalStorageKey } from '@/constants/localStorage';
+import { LOCAL_STORAGE_VERSIONS, LocalStorageKey } from '@/constants/localStorage';
 
 import { getLocalStorage, setLocalStorage } from '@/lib/localStorage';
 
 export interface LayoutState {
-  selectedTradeLayout: TradeLayouts;
+  selectedTradeLayout: {
+    version: string;
+    layout: TradeLayouts;
+  };
   isSidebarOpen: boolean;
 }
 
 const initialState = {
-  isSidebarOpen: true,
   selectedTradeLayout: getLocalStorage({
     key: LocalStorageKey.SelectedTradeLayout,
-    defaultValue: TradeLayouts.Reverse,
+    defaultValue: {
+      version: LOCAL_STORAGE_VERSIONS[LocalStorageKey.SelectedTradeLayout],
+      layout: TradeLayouts.Default,
+    },
   }),
+  isSidebarOpen: true,
 };
 
 export const layoutSlice = createSlice({
   name: 'Layout',
   initialState,
   reducers: {
+    setSelectedTradeLayout: (state: LayoutState, { payload }: PayloadAction<TradeLayouts>) => {
+      setLocalStorage({
+        key: LocalStorageKey.SelectedTradeLayout,
+        value: {
+          version: LOCAL_STORAGE_VERSIONS[LocalStorageKey.SelectedTradeLayout],
+          layout: payload,
+        },
+      });
+      state.selectedTradeLayout = {
+        version: LOCAL_STORAGE_VERSIONS[LocalStorageKey.SelectedTradeLayout],
+        layout: payload,
+      };
```

Comment:
nit -- maybe save this as a value so it doesn't need to b written twice?

Reply:
sounds gouda, will follow up in subsequent PR!!!